u2f-gae-demo/war/js/u2f-api.js

```diff
@@ -709,7 +717,7 @@ u2f.responseHandler_ = function(message) {
  * @param {number=} opt_timeoutSeconds
  */
 u2f.sign = function(appId, challenge, registeredKeys, callback, opt_timeoutSeconds) {
-  if (js_api_version === undefined) {
+  if (js_api_version === undefined && !u2f.isIOSChrome_()) {
```

Comment:
Is there any reason we can't add a request to get the API version to the iOS app? In general, we should strive to avoid using userAgent sniffing to determine which code path to follow, and use features instead. If the iOS app could return its API version, the rest of the code would be the same, no?